### DISCUSSION TITLE
3C小白怎麼選電腦螢幕

### DISCUSSION THREAD

**Discussion by Superlan8** [1101-GS4719-C/Python-Community-GS4719-C#1359] [General]
對於大學生來說，使用筆電或是平板等3C產品再平常不過了，有些人為了眼睛的舒適度，會選擇外接螢幕或是直接買桌機來保護眼睛，那對於一個新手3C玩家來說，電腦螢幕款式那麼多該怎麼選?這邊我幫各位列出幾項可以入門挑選該知道的事。

1. 你的桌子大小：在選螢幕第一要件是確認你的桌子大小及深度，一般而言，目前普遍的大小不外乎是24吋及27吋，桌子深度越大越深，你選擇的螢幕就能大一點，建議大家可以先去實體店模擬一下你桌子的深度，看這樣大小的距離會不會造成你眼睛的不適。
2. 你的螢幕用途：第二件重要的事是你要如何使用你的螢幕，假如你是用來玩遊戲的，那你可能就得挑選更新率高一點的螢幕，假如你是喜歡環繞曲面的感覺，曲面螢幕也是一種選擇。

這兩個基本條件都知道後，可以來認識一下螢幕的面板，現在大部分主流面板分成三大種類，VA、IPS、TN。

TN面板
        優點：便宜，TN面板是三種中成本最低的，基本上這種面板是用在一些較低階、入門的螢幕。
        缺點：色彩表現差，這也是TN面板的致命傷，還有另一個是可視角度小，假如你不是坐在正面看螢幕的話，亮度及色偏也都會有不小的差異。

VA面板
        優點：色彩還原度高，尤其是在黑色方面，這種面板可以展現很好的對比效果，但比起色彩，還是略遜IPS面板一籌。
        缺點：功耗比較高，且反應時間會較慢一點。

IPS面板
        優點：可視角度大，基本上可以接近180度，在色彩表現也非常的真實，動態畫質也表現得很好，無殘影的效能對於使用者的感受也非常好，同時，IPS面板相較於其他面板較薄，同時也會更加省電。
        缺點：漏光，儘管可視角度大，但光線穿透也會受影響，有時會產生漏光的現象。價格高，由於IPS各方面表現都不錯，自然也反映在價格上。黑色表現差，IPS在單純表現黑色的效果沒有VA面板好。

總結：
        TN面板表現真的不好，除非你真的完全不考慮色彩，不然我是不推薦的。
        如果你預算有限，我會建議如果只是看影片、電影或是文書處理選VA，有要玩遊戲的話選IPS。
        但假如你預算夠，我會建議都上IPS，畢竟IPS能應用的範圍最廣，通常高階螢幕也都是用IPS面板。

祝各位都能找你自己理想中的螢幕。

**Comment by brian431** [comment]
謝謝分享。我一直都認為在選配電腦時螢幕是非常重要的，無論是畫質、色彩，或刷新率，都有很大的影響。選對螢幕的話能讓使用電腦的體驗品質提升很多。個人就買了一個非常適合我的螢幕，雖然不便宜，不過是要整天盯著看的東西，還是買好一點的比較好。
